src/App.js

```diff
@@ -1,22 +1,19 @@
+import React, { PureComponent } from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>
-          I am Yannick Noel Aka
+const App = class extends PureComponent {
+  render() {
+    return (
+      <div className="App">
+        <div className="App-body">
+          <Calculator />
+        </div>
+      </div>
+    );
+  }
+};
```

Comment:
- [ ] Although this works, it is not necessary to use PureComponent here and assign the class to a const. This project requires the usage of a class base component to be familiar with that approach of creating components in React.

- [ ] Here is an example of a class-based component. Let's try to follow the pattern
```
// MyComponent.js
import React, { Component } from 'react';

class MyComponent extends Component {
  render() {
    return (
      <div>This is my component.</div>
    );
  }
}

export default MyComponent;
```

Reply:
Hey @EricMbouwe ,
Thank you for reviewing my work.
I completed all the requirements. 

Thank you.